scripts/auto_plot.bash

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#Run this script from a directory with an RTS solution and it will create the analyasis plots for you
+
+ATTEMPT_NUM=$(find . -mindepth 1 -maxdepth 1 -type d | wc -l)
+ATTEMPT_NUM=$(expr $ATTEMPT_NUM + 1)
+mkdir "attempt_number_${ATTEMPT_NUM}"
+
+cp BandpassCalibration_node0* ./attempt_number_${ATTEMPT_NUM} 
+cp DI_JonesMatrices_node0*.dat  ./attempt_number_${ATTEMPT_NUM}
+cp flagged_tiles.txt ./attempt_number_${ATTEMPT_NUM}
+cp flagged_channels.txt ./attempt_number_${ATTEMPT_NUM}
+
+cd ./attempt_number_${ATTEMPT_NUM}
+
+PLOT=`which plot_BPcal_128T.py`
+for i in `seq -w 1 24`; do
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4498157)